instructions.md

```diff
@@ -0,0 +1,13 @@
+This tool generates custom-voiced audiobook files from an imported ebook file. Please upload an ebook to begin the conversion process. Output files will be downloaded as a .zip archive.
```

Comment:
Make this a section in your README, not a separate file. To see what sections your README should contain, see my python packages notes on Slack.

Reply:
README update according to the python packages file. Instructions.md moved to resources folder.